rotkehlchen/exchanges/bittrex.py

```diff
@@ -355,17 +356,18 @@ def query_balances(self) -> Tuple[Optional[Dict[Asset, Dict[str, Any]]], str]:
                 )
                 continue
 
-            balance = {}
-            balance['amount'] = FVal(entry['total'])
-            balance['usd_value'] = FVal(balance['amount']) * usd_price
-            returned_balances[asset] = balance
-
+            amount = FVal(entry['total'])
+            usd_value = amount * usd_price
+            returned_balances[asset] = Balance(
+                amount=amount,
+                usd_value=usd_value,
+            )
             log.debug(
                 'bittrex balance query result',
                 sensitive_log=True,
                 currency=asset,
-                amount=balance['amount'],
-                usd_value=balance['usd_value'],
+                amount=amount,
```

Comment:
I've seen some exchanges (the oldest ones) that have these debug logs with kwargs when the balances are successfully requested. According to this morning talk, do you want me to either:
A. Keep them as they are and amend the kwargs to make them work (as in this PR).
B.  Remove the kwargs and inject the variables in the message using f-string.

Reply:
Hey thanks for asking. Maybe for this PR let's keep it simple. Doing A is good enough.

Perhaps we can in the future get rid of that log filter but still introduce kwargs logs in another way. Since kwargs in the logs are actually rather useful